### PR TITLE
Log operations

### PR DESCRIPTION
**Pull request by cicr99**
Implemented log opcodes behavior regarding:
- popping things from stack
- updating the memory consumption counter

### CODE REVIEW

File: tests/transpilation/log_zero_length.opcode [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PUSH1 0x00   # [0]
+PUSH1 0x80   # [128, 0]
```
Comment (murcake) [inline tests/transpilation/log_zero_length.opcode:2]: Cool, it's a nice idea to test the corner case

**Review by murcake — APPROVED** [review]
LGTM

File: cairo-src/evm/utils.cairo [hunk 2]
```diff
@@ -25,6 +25,15 @@ func ceil_div{range_check_ptr}(a, b) -> (res):
     end
 end
 
+func memory_expansion_for_range{range_check_ptr}(s, f, l) -> (res):
```
Comment (murcake) [inline cairo-src/evm/utils.cairo:28]: Perhaps, `update_msize` is a more suitable name? Together with more descriptive parameter names, it would make the function crystally clear
